testng-core/src/test/resources/parametertest/issue_581/parent_suite.xml

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
```

Comment:
dtd 1.1

Reply:
This never gets applied because we first default to loading the DTD that is available locally and the file name is fetched from `org.testng.xml.internal.Parser#TESTNG_DTD` (this I have updated to `1-1.dtd` in the code). So I think we should be fine even if we don't refer to the new version